test.py

```diff
@@ -38,8 +38,21 @@ def inference_args(parser, remaining_args):
 
     inf_parser.add_argument(
         '--residual',
-        action='store_true',
-        help='build a source for the mix minus estimated targets'
+        type=str,
+        default=None,
+        help='if provided, build a source with given name for the mix '\
+             'minus all estimated targets'
+    )
+
+    inf_parser.add_argument(
+        '--out',
+        type=str,
+        default=None,
+        help='if provided, must be a string containing a valid expression for '
+             'a dictionary, with keys the output target names, and values '
+             'a list of targets that are used to build it. For instance: '
+             '\'{\"vocals\":[\"vocals\"], \'accompaniment\':[\"drums\",'
+             '\"bass\",\"other\"]}\''
```

Comment:
I think this is a bit risky to pass via bash... will check for another option

Reply:
ok feel free to cha'ge that